src/utilities/bcl/templates/EnergyPlusMeasure/tests/test_energyplus_measure.py

```diff
@@ -1,11 +1,16 @@
 """Insert your copyright here."""
 
+import sys
 from pathlib import Path
 
 import openstudio
 import pytest
 
+CURRENT_DIR_PATH = Path(__file__).parent.absolute()
+sys.path.insert(0, str(CURRENT_DIR_PATH.parent))
 from measure import EnergyPlusMeasureName
+sys.path.pop(0)
+del sys.modules['measure']
```

Comment:
To be able to load several pytest files when --run-tests is happening, and because every measure is `measure.py`, we need del sys.modules['measures'] or it remembers the first one it found